independent-projects/bootstrap/maven-resolver/src/main/java/io/quarkus/bootstrap/resolver/maven/SecDispatcherImpl.java

```diff
@@ -44,11 +44,7 @@
     protected String _configurationFile = "~/.m2/settings-security.xml";
 
     public SecDispatcherImpl() {
-        try {
-            this._cipher = new DefaultPlexusCipher();
-        } catch (PlexusCipherException e) {
-            throw new IllegalStateException("Failed to init Security Dispatcher", e);
-        }
+        this._cipher = new DefaultPlexusCipher();
```

Comment:
Could we still keep try/catch catching a `Throwable` or something to stay backward compatible?

Reply:
Backward compatible? this actually was catching a specific checked exception `PlexusCipherException` and wrapped it in a runtime exception `IllegalStateException`, since the instantiation of `DefaultPlexusCipher` doesn't throw the checked exception anymore I don't see why trying to catch "all" exceptions (`Throwable`) would make this backward compatible.

Reply:
FTR:
- https://github.com/codehaus-plexus/plexus-cipher/commit/48b5a36bbace09d12f3f9ee24d516c88dd9a5e10#diff-edc9e74df359f6207f332996ff040bfe9bbea1f9e7800b71a0cae5f0c6346f7bL41
- https://github.com/codehaus-plexus/plexus-cipher/commit/48b5a36bbace09d12f3f9ee24d516c88dd9a5e10#diff-e34d3c221117a8ce5c9b679f4acdd7dede829586c9bc5cb907e099d9eb61aa96L77

Reply:
I quickly tried to break it but couldn't, so it's ok.

Reply:
According to @famod links above, I think it could break only if the `SHA-256` algorithm weren't available, which isn't the case since this algorithm is guaranteed to be supported in a JCA-compliant JVM